testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -3551,7 +3552,7 @@ public void testAddColumnConcurrently()
                             return Optional.of(columnName);
                         }
                         catch (Exception e) {
-                            RuntimeException trinoException = getTrinoExceptionCause(e);
+                            RuntimeException trinoException = getTrinoExceptionOrRootCause(e);
```

Comment:
Potential "expected" failures should always surface to the user as TrinoException.
The only case where we "allow" a RTE to be propagated to the query failure is when we have an internal bug.
That's possible when we have insufficient test coverage, but in this case, the test coverage is already there.
Instead of this change, ensure the possible failures always  come as TrinoException